### ISSUE TITLE
Version not showing properly via pip

### ISSUE THREAD

**Issue by snowman2** [conda-forge/datacube-feedstock#25]
```
$ conda list datacube
# packages in environment at /home/snowal/miniconda/envs/midas:
#
# Name                    Version                   Build  Channel
datacube                  1.8.0                      py_0    conda-forge
$ pip show datacube
Name: datacube
Version: 0.0.0
Summary: An analysis environment for satellite and other earth observation data
Home-page: https://github.com/opendatacube/datacube-core
Author: Open Data Cube
Author-email: None
License: Apache License 2.0
Location: ~/miniconda/envs/odc/lib/python3.6/site-packages
Requires: dask, python-dateutil, jsonschema, rasterio, click, cachetools, distributed, netcdf4, numpy, psycopg2, toolz, cloudpickle, shapely, pyyaml, sqlalchemy, xarray, pyproj, affine, lark-parser
Required-by: 
```

**Comment by snowman2** [comment]
cc: @Kirill888

**Comment by snowman2** [comment]
The version shows up fine when I `pip install datacube`.

**Comment by Kirill888** [comment]
Thanks for pointing this out @snowman2 I'm not super familiar with conda, I guess this is a missing build dependency for `<something>_scm`, I'll have a look

**Comment by Kirill888** [comment]
I think issue is this: installing datacube from source archive is fine, but building a wheel from source archive results in 0.0.0 version, and conda seems to build wheel from source, then install wheel resulting in broken version

**Comment by Kirill888** [comment]
Pretty sure this is fixed in `datacube=1.8.0=py_1`.
Build step needed `setuptools_scm` and `toml` for `pyproject.toml` handling.

**Comment by snowman2** [comment]
Thanks 👍
